### PR TITLE
Mozhi Malayalam - Autogenerated

### PR DESCRIPTION
**Pull request by beniza**
From this update onwards the keyboard rules are generated directly from the Spec definition sheet:  https://docs.google.com/spreadsheets/d/1uBtNtaLPESjjdvv3X__qHkBGoO9UfTIjGOIdcgAgxwA as per the Mozhi 2 specs listed in: https://sites.google.com/site/cibu/mozhi2

In addition to the above this PR will:
  add a new Layout illustration
  change the copyright ownership

### CODE REVIEW

**Comment by keyman-server** [comment]
This pull request is from an external repo and will not automatically be built. The build must still be passed before it can be merged. Ask one of the team members to make a manual build of this PR.

**Comment by DavidLRowe** [comment]
FYI: we now only require the date (or date range) in the copyright statement in LICENSE.md. Any other copyright statements may omit the date. This potentially reduces the number of files that need to change for a new version of the keyboard.

The .kps file needs some additional files.
On the Files tab: 
- include the .kvk file from build folder
- include welcome.htm (and all files it references) and readme.htm

This will include the on-screen keyboard (.kvk), the file displayed just prior to installation (readme.htm) and the offline help (welcome.htm and any files it references).

Were you planning to include the fonts (which are in the assets folder) in the keyboard package? If so, they should be referenced in the .kps file as well.

On the Details tab:
- Consider removing current version (3.2) and ticking the "Package version follows keyboard version" box. (This means that, for a new version of the keyboard, you wouldn't have to touch this file just to change the version number.
- Consider removing date (2021-2022) from Copyright field. (Similar reasoning as the preceding suggestion.)

I'm not sure what adding "assets" to the .gitignore file will do. Does that mean any changes to the assets folder won't be propagated to the repository?

minor point: readme.htm has a typo in the title: "Keybaord" should be "Keyboard". Also "It's" should be "Its" in the second sentence of the second paragraph.

**Comment by beniza** [comment]
> Has there been an agreement that the copyright holder can change? I don't think we can reassign the copyright without the original author's permission.

The current owner, Junaid P. V., has agreed to the copyright ownership change and his mail has been forwarded to you. Do let me know if there's anything else to be done in this regard.

**Comment by beniza** [comment]
@DavidLRowe 
Thanks for the detailed feedback. I've made those changes and I hope that I've fixed them all. Your attention to the details is commendable 👏.

**Comment by DavidLRowe** [comment]
@beniza Will this PR address #1445 and #1659 ?

Let me know when you're ready for a review. Thanks!

**Comment by beniza** [comment]
#1445 is fixed. No whitespaces around the `key-text`s
#1659 Akhil [offered to write a detailed documentation](https://community.software.sil.org/t/em-dash-and-en-dash-in-mozhi-keyboard/5472/6) and [I've requested](https://github.com/keymanapp/keyboards/issues/1659#issuecomment-1051094371) him to do so. I've included a new keyboard layout file with most of the commonly used characters. A link to the Mozhi spec that provides a complete coverage of characters are also included in this PR.

**Comment by beniza** [comment]
@DavidLRowe please do a review and let me know if it's good to go.

**Comment by DavidLRowe** [comment]
Looking very good. Just two things for the package file (.kps):
- The path to the .kvk file is incorrect. (Needs `..` before `build` to get to the build folder.)
- On the Details tab, under "Optional Information", use the pull-down menu next to "Readme file:" to select the "readme.htm" file.

**Comment by DavidLRowe** [comment]
Those changes look good. 

Sorry I didn't catch this before, but the version number you have (3.2.22.05.09.16) doesn't fit the pattern (of two or three numbers separated by dots, that is, 1.0 for major.minor, or 1.2.3 for major.minor.patch). This could affect the ability of Keyman to notify users of updated versions. 

Would it be possible to change it to 3.2? If you change it in the .kmn file, the change should get picked up in the .kps file.

Thanks!

File: release/m/mozhi_malayalam/mozhi_malayalam.kpj [hunk 1]
```diff
@@ -4,32 +4,31 @@
...
       <Filename>mozhi_malayalam.kmn</Filename>
       <Filepath>source\mozhi_malayalam.kmn</Filepath>
-      <FileVersion>2.1</FileVersion>
+      <FileVersion>3.2.22.05.09.16</FileVersion>
       <FileType>.kmn</FileType>
       <Details>
         <Name>Malayalam Mozhi</Name>
-        <Copyright>Copyright © 2017-2020 Junaid P. V.</Copyright>
-        <Message>A complete rebuild of Mozhi keyboard updated for Mozhi 2.0</Message>
+        <Copyright>© Cibu C. J.</Copyright>
```
Comment (beniza) [inline release/m/mozhi_malayalam/mozhi_malayalam.kpj:19]: Junaid has **_transferred the ownership_** to Cibu vide his mail regarding the same.

**Comment by beniza** [comment]
@DavidLRowe I've change the keyboard version back to 3.2. (btw, you didn't miss that one earlier. it got changed during my last update. I reverted it now)

**Review by DavidLRowe — APPROVED** [review]

**Comment by beniza** [comment]
Thank you David, and sorry for all the trouble.
Glad to see that the autogenerated code passes the tests.

**Pull request merged**